Resources/Audio/_Goobstation/Voice/Moth/attributions.yml

```
@@ -0,0 +1,4 @@
+- files: ["moth_flap.ogg"] 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)